openpype/hosts/maya/plugins/load/load_arnold_standin.py

```diff
@@ -84,6 +96,10 @@ def load(self, context, name, namespace, options):
             sequence = is_sequence(os.listdir(os.path.dirname(self.fname)))
             cmds.setAttr(standin_shape + ".useFrameExtension", sequence)
 
+            cmds.setAttr(standin_shape + ".dso", path, type="string")
+            matching_fps = get_fps(os.listdir(os.path.dirname(self.fname)))
+            cmds.setAttr(standin_shape + ".abcFPS", matching_fps)
```

Comment:
```suggestion
            fps = float(version["data"]["fps"])
            cmds.setAttr(standin_shape + ".abcFPS", fps)
```
you can get the fps from the published ass/abc from the version data.

Reply:
Note that technically a version **could** be lacking fps data, right? E.g. a publish from standalone publisher? (I suspect that doesn't know anything about its FPS?)

Reply:
Something like this:(pesudo code only)
```
            has_published_fps = version["data"]["fps"]
            matching_fps = None
            if has_published_fps:
                matching_fps = float(version["data"]["fps"])
            else:
                matching_fps = get_fps(os.listdir(os.path.dirname(self.fname)))
            cmds.setAttr(standin_shape + ".abcFPS", matching_fps)
```